nav2_system_tests/src/planning/planner_tester.cpp

```diff
@@ -36,17 +36,15 @@ namespace nav2_system_tests
 
 PlannerTester::PlannerTester()
 : Node("PlannerTester"), map_publish_rate_(100s), map_set_(false), costmap_set_(false),
-  using_fake_costmap_(true), costmap_server_running_(false), trinary_costmap_(true),
+  using_fake_costmap_(true), trinary_costmap_(true),
   track_unknown_space_(false), lethal_threshold_(100), unknown_cost_value_(-1),
   testCostmapType_(TestCostmap::open_space), spin_thread_(nullptr)
 {
-  // The client used to invoke the services of the global planner (ComputePathToPose)
-  planner_client_ = rclcpp_action::create_client<nav2_msgs::action::ComputePathToPose>(
-    this->get_node_base_interface(),
-    this->get_node_graph_interface(),
-    this->get_node_logging_interface(),
-    this->get_node_waitables_interface(),
-    "ComputePathToPose");
+  // The navfn wrapper
+  auto state = rclcpp_lifecycle::State();
+  planner_tester_ = std::make_unique<NavFnPlannerTester>();
```

Comment:
With the multi-node networking aspects removed from this test, I think it would make sense to move this whole test to an "integration" test in the navfn package.

Reply:
Agreed

Reply:
Thinking about the implementation details, I think its probably not worth doing right now since we have the `COLCON_IGNORE` on this package (and this test has 101 trials) would take awhile for a typical run. We may want to leave it here for now